build-support/flake8/.flake8

```diff
@@ -8,3 +8,5 @@ extend-ignore:
   W503,  # line break before binary operator  (conflicts with Black)
   PB10,  # Bad class attribute (enable once fixed)
   PB13,  # `open()` not within a context manager (possibly enable once fixed)
+  NIC002, # Implicitly concatenated string literals over multiple lines
+  NIC102, # Implicitly concatenated bytes literals over multiple lines
```

Comment:
It seems like the majority of these issues actually have to do with multi-line strings. Does this still end up catching anything useful for us with multi-line disabled? Maybe the dangling comma case...?

I think that in general we do still want to use multi-line implicit concat.

Reply:
yes, but I think those are fine to ignore since they are in multiple lines which makes the code more readable.


We had cases that this plugin caught (not bugs, but still): https://github.com/pantsbuild/pants/pull/14119/